hack/verify-codegen.sh

```diff
@@ -3,7 +3,15 @@
 if [ "$IS_CONTAINER" != "" ]; then
   set -x
```

Comment:
Just remove the `set -x` and `set +x` instead of going through the machinations of checking the return codes.

Reply:
i added `e` instead